tox.ini

```diff
@@ -74,6 +74,17 @@ commands =
     pytest {tty:--color=yes} --cov=snapcraft --cov-report=xml:results/coverage-{env_name}.xml --junit-xml=results/test-results-{env_name}.xml tests/unit {posargs}
     pytest {tty:--color=yes} --junit-xml=results/legacy-test-results-{env_name}.xml tests/legacy {posargs}
 
+[testenv:test-all-py310]
+base = testenv
+description = Run all tests with the interpreter used in the snap
```

Comment:
shouldn't we use the requirements files? To match the dependency versions

Reply:
yeah, I made a back and forth copy paste mistake